aksel.nav.no/website/components/token-docs/toolbar/SearchField.utils.ts

```diff
@@ -0,0 +1,30 @@
+import Fuse from "fuse.js";
+import { tokens } from "@navikt/ds-tokens/token_docs";
+import { TokenForDocumentationT } from "../../types/tokens";
+
+function tokenIndex(_tokens: TokenForDocumentationT[]) {
+  return new Fuse(_tokens, {
+    threshold: 0.1,
+    keys: [
+      { name: "name", weight: 0.7 },
+      { name: "group", weight: 0.3 },
+      { name: "category", weight: 0.3 },
+      { name: "comment", weight: 0.3 },
+      { name: "type", weight: 0.3 },
+      { name: "role", weight: 0.3 },
+      { name: "modifier", weight: 0.3 },
+    ],
+    shouldSort: false,
+    ignoreLocation: true,
```

Comment:
Ignoring location to accept hits anywhere in the texts. We have no texts longer than two lines, so there are no "hits at the end of the page"-cases.